code/datums/tutorial/marine/basic_marine.dm

```diff
@@ -0,0 +1,208 @@
+/datum/tutorial/marine/basic
+	name = "Marine - Basic"
+	desc = "A tutorial to get you acquainted with the very basics of how to play a groundside marine role."
+	tutorial_id = "marine_basic_1"
+	tutorial_template = /datum/map_template/tutorial/s8x9/no_baselight
+	/// How many items need to be vended from the clothing vendor for the script to continue, if something vends 2 items (for example), increase this number by 2.
+	var/clothing_items_to_vend = 8
+	/// How many items need to be vended from the gun vendor to continue
+	var/gun_items_to_vend = 2
+
+// START OF SCRIPTING
+
+/datum/tutorial/marine/basic/start_tutorial(mob/starting_mob)
+	. = ..()
+	if(!.)
+		return
+
+	var/obj/item/device/flashlight/flashlight = new(loc_from_corner(2, 3))
+	flashlight.anchored = TRUE
+	flashlight.set_light_power(4)
+	flashlight.set_light_range(12)
+	flashlight.icon = null
+	flashlight.set_light_on(TRUE)
+	add_to_tracking_atoms(flashlight)
+
+	init_mob()
+	message_to_player("This is the tutorial for marine rifleman. Leave the cryopod by pressing <b>W</b> or <b>D</b> to continue.")
+	update_objective("Exit the cryopod by pressing W or D.")
```

Comment:
you may want to consider implementing a symbol proc to fetch keybinds just so you don't run into some sort of issue where someone has redefined their keybinds and the buttons we're telling them to press don't do what they're supposed to do.